tests/ontopy_tests/test_graph.py

```diff
@@ -77,7 +77,7 @@ class hasPartRenamed(owlready2.ObjectProperty):
     graph.save(tmpdir / "testonto.png")
 
 
-def test_emmo_graphs(emmo: "Ontology", tmpdir: "Path") -> None:
+def test_emmo_graphs(emmo: "Ontology", repo_dir: "Path") -> None:
```

Comment:
Please also rename the variable name in the documentation below accordingly

Reply:
Well spotted. I just changed it to repo_dir temporarily to see the actual output.I will change back to tmpdir.